docker/install/ubuntu_install_sphinx.sh

```diff
@@ -20,14 +20,14 @@ set -e
 set -u
 set -o pipefail
 
-# NOTE: install docutils < 0.17 to work around https://github.com/readthedocs/sphinx_rtd_theme/issues/1115
+# pin sphinx<7 because https://github.com/readthedocs/sphinx_rtd_theme/issues/1465
 pip3 install \
     autodocsumm \
-    "commonmark>=0.7.3" \
-    "docutils>=0.11,<0.17" \
+    commonmark \
+    docutils \
     Image \
     matplotlib \
-    sphinx==4.2.0 \
+    "sphinx<7" \
```

Comment:
It would be great if you could pin to a specific version (via ==x.y.z) rather than `<7.0`

Reply:
unpinned sphinx version since docs build works